source/plugin/Assets/GoogleMobileAds/Platforms/iOS/GoogleMobileAdsClientFactory.cs

```diff
@@ -69,7 +72,14 @@ public IRewardedAdClient BuildRewardedAdClient()
       {
         return new GoogleMobileAds.iOS.RewardedAdClient();
       }
-      return new GoogleMobileAds.Common.RewardedAdDummyClient();
+      else if (Application.platform == RuntimePlatform.OSXEditor || Application.platform == RuntimePlatform.WindowsEditor)
```

Comment:
Do you know why this is in the interstitial client method? Why isnt there a public IRewardedClient BuildRewardedClient()?